packages/inspector/src/add/add-trigger.ts

```
@@ -0,0 +1,154 @@
+  ).pikkuFuncName
+
+  if (!nameValue) {
+    return
+  }
+
+  // --- resolve middleware ---
+  const middleware = resolveMiddleware(state, obj, tags, checker)
+
+  // --- track used functions/middleware for service aggregation ---
+  state.serviceAggregation.usedFunctions.add(pikkuFuncName)
+  extractWireNames(middleware).forEach((name) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)